schema/appmodel/PDS.schema.xml

```diff
@@ -170,16 +173,21 @@
   <attribute name="offset" description="Pedestal for the channel" type="u16" range="0..2700" init-value="0" is-not-null="yes"/>
   <attribute name="trim" description="trim value for the channel" type="u16" range="0..4095" init-value="0" is-not-null="yes"/>
   <attribute name="channel_id" type="u8" range="0..39,100" init-value="100" is-not-null="yes"/>
+  <attribute name="stream_id" description="In Daphne v3 we might need to link a channel to a specific link ID, and this is the hook for the operation" type="s8" init-value="-1" is-not-null="yes"/>
  </class>
 
- <class name="DaphneV2ControllerModule">
+ <class name="DaphneControllerModule">
   <superclass name="DaqModule"/>
-  <attribute name="address" description="ip address of the board" type="string" is-not-null="yes"/>
   <relationship name="daphne_conf" class-type="DaphneConf" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
   <relationship name="board_conf" class-type="DaphneV2BoardConf" low-cc="one" high-cc="one" is-composite="yes" is-exclusive="yes" is-dependent="yes"/>
-  <method name="get_slot" description="">
-   <method-implementation language="C++" prototype="auto get_slot() const { return get_board_conf()-&gt;get_slot_id(); }" body="BEGIN_HEADER_PROLOGUE&#xA;#include &lt;appmodel/DaphneV2BoardConf.hpp&gt;&#xA;END_HEADER_PROLOGUE"/>
-  </method>
+ </class>
+
+ <class name="DaphneV2ControllerModule">
```

Comment:
What is the purpose of the `DaphneV2ControllerModule` and `DaphneV3ControllerModule` since they are identical to the parent class?